rpcs3/rpcs3qt/qt_utils.cpp

```diff
@@ -238,11 +238,12 @@ namespace gui
 		{
 			// get Icon for the gs_frame from path. this handles presumably all possible use cases
 			const QString qpath = qstr(path);
-			const std::string path_list[] = { path, sstr(qpath.section("/", 0, -2)), sstr(qpath.section("/", 0, -3)) };
+			const std::string path_list[] = { path, sstr(qpath.section("/", 0, -2, QString::SectionIncludeTrailingSep)),
+											  sstr(qpath.section("/", 0, -3, QString::SectionIncludeTrailingSep)) };
 
 			for (const std::string& pth : path_list)
 			{
-				if (!fs::is_dir(pth))
+				if (pth.empty() || !fs::is_dir(pth))
```

Comment:
is_dir should be false on empty strings

Reply:
Sure but if we know in advance why even bother it?

Reply:
it seems redundant to me

Reply:
You might as well change every single is_dir occasion to always have that condition added.
But why? The method was created with that in mind.

Reply:
Fair